src/Lightest.Data.Mongo/Models/Mongo.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+namespace Lightest.Data.Mongo.Models
+{
+    public class Mongo
+    {
```

Comment:
Either now or in following PRs, create three constructors:
1. `UploadData(string code)` - should generate new Guid and set `Code` field.
2. `UploadData(byte[] project)`  - should generate new Guid and set `Project` field.
3. `UploadData()` - empty body.